managed/ui/src/components/config/Storage/StorageConfiguration.js

```diff
@@ -183,13 +220,114 @@ class StorageConfiguration extends Component {
     this.props.fetchCustomerConfigs();
   }
 
+  /**
+   * This method will enable edit options for respective
+   * backup config.
+   */
+  onEditConfig = () => {
+    this.setState({ enableEdit: true });
+  }
+
+  /**
+   * This method will disable the edit input fields.
+   */
+  disableEditFields = () => {
+    this.setState({ enableEdit: false });
+  }
+
+  /**
+   * This method will help to disable the backup storage
+   * location field.
+   * 
+   * @param {string} fieldKey Input Field Id.
+   * @returns Boolean.
+   */
+  disableInputFields = (fieldKey, enableEdit, activeTab) => {
+    const tab = activeTab.toUpperCase();
+    return !enableEdit || fieldKey === `${tab}_BACKUP_LOCATION` ? true : false
+  };
+
+  /**
+   * This method will help us to setup the initial value props
+   * to the redux form.
+   * 
+   * @param {string} activeTab Current Tab.
+   * @param {Array<object>} configs Backup config Data.
+   */
+  setInitialConfiValues = (activeTab = "nfs", configs) => {
```

Comment:
- typo in method name
- why `nfs` is the default value?

Reply:
Done. I was passing `props.activeTab` as an argument and there's no default value for this so I assigned the default val. But take care of that.